README.md

```diff
@@ -326,6 +319,8 @@ This project would not be where it is today without the generous help provided b
 
 *   [Kristoffer Roupé](https://github.com/kitofr): Command line globbing
 
+*   [Alexander Popov](https://github.com/AlexWayfer): Daemon mode, many small fixes and improvements
+
 This gem was initially inspired by [Tom Lieber's blogg posting](http://alltom.com/pages/detecting-file-changes-with-ruby).
```

Comment:
I'll add link to [the last live archive version](http://web.archive.org/web/20120208094934/http://alltom.com/pages/detecting-file-changes-with-ruby).